bulma/helpers-rtl.scss

```diff
@@ -0,0 +1,184 @@
+.has-vertically-centered-content {
```

Comment:
I guess we could load the initial helpers and then override the left / right ones

It would be easier to maintain in the future.

Reply:
that's a good idea. i will change it that way. 
thanks